cue/data/gen.cue

```diff
@@ -209,8 +218,10 @@ Family: scuemata.#Family & {
                     // TODO tighter constraint
                     timeShift?: string
 
-                    // The values depend on panel type
-                    options: {...}
+                    // The allowable options are specified by the panel plugin's
+                    // schema.
+                    // FIXME same conundrum as with the closed validation for fieldConfig.
+                    options: {}
```

Comment:
In the first pass at this PR, i neglected to disable openness on this `options` field. Doing so revealed a number of other issues - missing declarations in schema that are fixed by other changes in the recent commits.